tests/components/emulated_hue/test_hue_api.py

```diff
@@ -240,10 +240,10 @@ def test_put_light_state_script(self):
             level)
 
     def test_put_light_state_media_player(self):
-        """Test turning on media player and setting volume."""
-        # Turn the music player off first
+        """Test setting media player volume."""
```

Comment:
this is updated doc to the change trisk suggested. so the test is not turning off the media player and testing that changing the volume will auto turn it on as it will not.